### ISSUE TITLE
nhl api broken

### ISSUE THREAD

**Issue by Psybernaut** [ephro/nhl-notifier#5]
Looks like the api link is no longer working
https://statsapi.web.nhl.com/api/v1/schedule?expand=schedule.linescore

I think there's an alternative, but I can't seem to find the right syntax.
https://api-web.nhle.com/v1/schedule/2023-11-20

It also seems to want a date, but that may just be me not knowing enough.
This might be helpful...https://github.com/ma-barbour/nhl_api_functions

**Comment by sirpeep** [comment]
Bump, this would be a pretty big fix

**Comment by ephro** [comment]
just pushed an update. Only two games today so it's hard to test it all, but let me know if you find anything. It looks like it's working so far.

Power plays could be broken

**Comment by Psybernaut** [comment]
Awesome! I'll check it once there's a Bruins game on.

**Comment by Psybernaut** [comment]
Welp, it looks like the IFTTT Webhooks option is now only a Pro feature.
Since I don't have a Pro account I can't test it.

**Comment by ephro** [comment]
Do you have some other endpoint you want to hit?

**Comment by Psybernaut** [comment]
I'm not sure what you mean by endpoint.
I was using the code to activate my nanoleaf lights when a goal is scored.
I managed to use the app notification on my phone to activate them, but it only works if my phone is in use so not ideal.
If there's an alternative to IFTTT then I'd use that.

**Comment by sirpeep** [comment]
> If there's an alternative to IFTTT then I'd use that.

What I would do is use home assistant, it can be kinda hard to setup but it works great and have had no issues so far. It has webhooks built-in. Or you can try zapier but it might cost money.

**Comment by ephro** [comment]
Yeah, I have Home Assistant too, and have been thinking of using it as in
input. What do you run HA on? Would a repo where you could run it on the
same server work for you?

**Comment by BigGreenEgger** [comment]
Hi guys.  I've been tinkering with the code for a bit.  I will upload my version shortly.  It is working for goal and power play notifications but there are a few minor bugs.  I have LIFX bulbs so I called their API directly because I didn't want to pay the monthly fee for IFTTT either.  I've stubbed the IFTTT calls so you can still use the code if you want.  I haven't coded in years so please be kind :)

**Comment by Psybernaut** [comment]
I'll probably setup Home Assistant at some point, but way too busy too get that running now.
A repo on the same server might work for me.

**Comment by sirpeep** [comment]
> What do you run HA on?

I run it on a oracle cloud VM for free, they give you a 1GB ram lifetime VM. I use it with the rest commands integration to allow for direct communication with the Govee API.

**Comment by BigGreenEgger** [comment]
Just uploaded my forked code.  Supports LIFX bulbs and works for goals and power plays.  @ephro Have a look and see what you think.  I documented a few minor issues and the delay code was giving me errors.  Comments in the code as well.  Hope this helps and someone can modify this to use something more generic than specific LIFX API calls.  IFTTT code is still stubbed if people want to use it.  Enjoy!
